### ISSUE TITLE
Testing: Linked resources pagination

### ISSUE THREAD

**Issue by jimsafley** [omeka/omeka-s#2227]
The [linked-resources-async](https://github.com/omeka/omeka-s/tree/linked-resources-async) branch iimproves the usability of the "Linked resources" section of resource pages. Before, the linked resource pages updated by loading the entire page. Now, the linked resource pages update by loading them in place, asynchronously. In doing this, we avoid the jarring and unexpected effect of page loads while paging through the linked resources.

To test this, find an item with many linked resources (more than the configured per page). Clicking on the linked resources pagination should update the linked resources without loading the entire page.

**Comment by zerocrates** [comment]
This is merged to develop now, so should be looked at there, not in the branch.

**Comment by allanaaa** [comment]
Update: 2 seems to be true in some themes, but not all of them. 

https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-second-site/item-set/4147 (Lively) is showing the LR table.

https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-third-site/item-set/4255 (Cozy) is not.

**Comment by allanaaa** [comment]
My second site, in the Lively theme, is showing "1-25" for the items in the item set, but the site is set to 6 results per page. It is accurately showing 6 results per page in the table. 

https://dev.omeka.org/amayer/amayer-s/omeka-s/s/my-second-site/item-set/4147

**Comment by jimsafley** [comment]
> Paging through the LR is accurately only refreshing that portion of the page. Is "25" a hardcoded value? How hard is it for site admin to change that?

Yes, it's hard coded. I'm pretty sure I can set it to the site's "results per page" and fall back on the installation's "results per page." Is that okay? I prefer not to add another setting.

> Paging through the items table refreshes the entire page, including taking the LR table back to its page 1. Which now feels a bit heavy-handed compared to how the LR table paginates so nicely, but, it's fine.

I agree. It's something that we'll probably address at a later date.

> When you click on any item link in the LR table, it does not actually link to the resource. All of the links are just the URL of the same page you're already on (the item set).

Thanks for pointing out this oversight. Not sure how it escaped my notice. Anyway, I pushed a change that should fix it.

> And, it appears that the LR table is itself private - only viewable to logged-in users. This is true of items and item sets.

I can't think of a way that this feature would do this. Usually when things are theme-specific, it's because the theme itself overwrites or modifies the behavior in some way.

**Comment by allanaaa** [comment]
Pulling from the site setting sounds ideal, thanks. 

Seems like we might have to do a round of theme updates since this appears very inconsistent. I'll check everything thoroughly in each theme and log issues in their repos.

**Comment by allanaaa** [comment]
And you've set the LR table's item links to open in new tabs/windows, which is fine, but isn't how we normally do things. I'm fine with leaving it as is but thought I'd mention it.

**Comment by jimsafley** [comment]
[Lively](https://github.com/omeka-s-themes/lively/blob/master/view/common/linked-resources.phtml) and [Freedom](https://github.com/omeka-s-themes/freedom/blob/master/view/common/linked-resources.phtml) overwrite `common/linked-resources.phtml`.

> And you've set the LR table's item links to open in new tabs/windows, 

The linked resources table would revert to its first page if it opened in the same tab and the user clicked to go back.
